web_monitoring/differs.py

```diff
@@ -51,11 +52,20 @@ def pagefreezer(a_url, b_url):
     # Just send PF the urls, not the whole body.
     # It is still useful that we downloaded the body because we are able to
     # validate it against the expected hash.
-    return web_monitoring.pagefreezer.compare(a_url, b_url)
+    obj = web_monitoring.pagefreezer.PageFreezer(a_url, b_url)
```

Comment:
The class `PageFreezer` does some expensive operations on `__init__` (parse_to_df, use_filter) whose results don't matter to us here and just get thrown away. I think it was more intended for interactive use, which is why I factored out the underlying operations into functions that can be called individually.

The question of whether to result the whole response or just the 'result' part is debatable. Some of the metadata in the response (like PageFreezer's claim of how long it took to compute) could be interesting, so I judged that it's probably better not to throw it away in this layer. But reasonable people could disagree on that. :- )